aws/ami/windows/windows.pkr.hcl

```diff
@@ -23,13 +23,14 @@ source "amazon-ebs" "windows_ebs_builder" {
     device_name           = "/dev/sda1"
     volume_size           = 64
   }
-  source_ami     = "${data.amazon-ami.windows_root_ami.id}"
-  region         = "us-east-1"
-  ami_regions    = ["us-east-1", "us-east-2"]
-  user_data_file = "user-data-scripts/bootstrap-winrm.ps1"
-  winrm_insecure = true
-  winrm_use_ssl  = true
-  winrm_username = "Administrator"
+  source_ami      = "${data.amazon-ami.windows_root_ami.id}"
+  region          = "us-east-1"
+  ami_regions     = ["us-east-1", "us-east-2"]
+  user_data_file  = "user-data-scripts/bootstrap-winrm.ps1"
+  winrm_insecure  = true
+  winrm_use_ssl   = true
+  winrm_username  = "Administrator"
+  skip_create_ami = var.skip_create_ami
```

Comment:
Also added this option to skip AMI creation by default so that we don't have a bunch of test amis laying around